Readme.md

```diff
@@ -144,6 +145,28 @@ If this is your first time using Carthage in the project, you'll need to go thro
 
 > NOTE: At this time, Carthage does not provide a way to build only specific repository submodules. All submodules and their dependencies will be built with the above command. However, you don't need to copy frameworks you aren't using into your project. For instance, if you aren't using `ReactiveSwift`, feel free to delete that framework along with `ReactiveMoya` from the Carthage Build directory after `carthage update` completes. Or if you are using `ReactiveSwift` but not `RxSwift`, then `RxMoya`, `RxTest`, `RxCocoa`, etc. can safely be deleted.
 
+### Accio
+
+Add the following to your Package.swift:
```

Comment:
I feel like this entire thing should just be "Do the same as SPM but use `accio update` instead". Any reason the repetition is needed?

Reply:
You're right about that. We could also combine both into one section names "SwiftPM & Accio" or would you rather keep it separate?

Reply:
I vote for separation, mainly because `SwiftPM & Accio` _may_ be confusing for users looking just for the SwiftPM instructions. 

So basically a 👍 to @freak4pc suggestion

Reply:
I just changed the installation instructions for Accio to refer to the SwiftPM section, like suggested.